registry.js

```diff
@@ -118,7 +118,6 @@ class CustomAttributeRegistry {
     }
     // Attribute was removed
     else if(newVal == null) {
-      inst.value = newVal;
```

Comment:
The reasoning here is that `this.value`  will show the last value that the Custom Attribute had, when being disconnected. Otherwise, the value will be gone. The user could infer that the actual DOM attribute is `null` if `disconnectedCallback` is being called.

So it's more of like `null` doesn't exist in the eyes of a Custom Attribute, instead it just gets disconnected. The `null` is more of a value in the eyes of a element (f.e. Custom Element).

In my use cases, I found it useful to know what the last value was during disconnected.